sys/compat/linux/linux_misc.c

```diff
@@ -29,13 +29,16 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "opt_posix.h"
```

Comment:
Why is this header needed?

Code you changed previously wrapped sys_kmq_ calls into #ifdefs.  Either we need to be same careful there, or I do not see a use for the option.h